sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClientOptions.cs

```diff
@@ -207,6 +213,44 @@ public int PrefetchCount
             }
         }
 
+        /// <summary>
+        ///   The desired amount of time to allow between load balancing verification attempts.
+        /// </summary>
+        ///
+        /// <remarks>
+        ///   Because load balancing holds less priority than processing events, this interval
+        ///   should be considered the minimum time that will elapse between verification attempts; operations
+        ///   with higher priority may cause a minor delay longer than this interval for load balancing.
+        /// </remarks>
+        ///
+        public TimeSpan LoadBalancingUpdateInterval
+        {
+            get => _loadBalancingUpdateInterval;
+
+            set
+            {
+                Argument.AssertNotNegative(value, nameof(LoadBalancingUpdateInterval));
+                _loadBalancingUpdateInterval = value;
+            }
+        }
+
+        /// <summary>
+        ///   The desired amount of time to consider a partition owned by a specific event processor
+        ///   instance before the ownership is considered stale and the partition eligible to be requested
```

Comment:
```suggestion
        ///   instance before the ownership is considered stale and the partition becomes eligible to be requested
```

Reply:
Applied manually to avoid confusing my rebase.